purchases/src/main/kotlin/com/revenuecat/purchases/AppLifecycleHandler.kt

```diff
@@ -3,17 +3,32 @@ package com.revenuecat.purchases
 import androidx.lifecycle.DefaultLifecycleObserver
 import androidx.lifecycle.LifecycleOwner
 
+@SuppressWarnings("EmptyFunctionBlock")
 internal class AppLifecycleHandler(
     private val lifecycleDelegate: LifecycleDelegate,
 ) : DefaultLifecycleObserver {
-
     override fun onStart(owner: LifecycleOwner) {
         lifecycleDelegate.onAppForegrounded()
     }
 
     override fun onStop(owner: LifecycleOwner) {
         lifecycleDelegate.onAppBackgrounded()
     }
+
+    // Some functions are implemented and left empty to prevent issues with Java default interfaces
```

Comment:
idk if it's worth it, but we could add a todo to remove when unity updates gradle requirements...